config/tests/fixtures/minimal_alone_with_genesis.toml

```
@@ -0,0 +1,6 @@
+extends = "base.toml"
+
+[genesis]
+file = "./empty_ok_genesis.json"
```

```suggestion
path = "./empty_ok_genesis.json"
```

I already mentioned this makes more sense to me